net-misc/batctl/batctl-2021.0.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 2020 Gentoo Authors
```

Comment:
Should be updated. Repoman does this automatically for you if you use that to commit.